pyccel/codegen/printing/ccode.py

```diff
@@ -63,6 +63,7 @@
                   ('complex',4) : 'float complex',
                   ('int',4)     : 'int',
                   ('int',8)     : 'long',
+                  ('pyobject', 0) : 'PyObject',
```

Comment:
If we add a getter in the class for collecting objects from this dictionary then we can extend the function in the cwrapper. That also means that we only need to write the try/except once in the getter rather than every time we use the dictionary